src/Raven.Server/Smuggler/Documents/SmugglerExporter.cs

```diff
@@ -21,27 +24,31 @@ public class SmugglerExporter
         public long? StartRevisionDocumentsEtag;
         public int? RevisionDocumentsLimit;
 
+        public DatabaseExportOptions Options;
         public DatabaseItemType OperateOnTypes;
 
-        public SmugglerExporter(DocumentDatabase database)
+        public SmugglerExporter(DocumentDatabase database, DatabaseExportOptions options = null)
         {
             _database = database;
             OperateOnTypes = DatabaseItemType.Indexes | DatabaseItemType.Transformers
                 | DatabaseItemType.Documents | DatabaseItemType.RevisionDocuments | DatabaseItemType.Identities;
+            Options = options;
```

Comment:
Options can be null, but later I do not see any NRE checks. Probably if they are null then you want to create new instance with default values.